libraries/AP_Baro/AP_Baro.cpp

```diff
@@ -225,6 +228,32 @@ const AP_Param::GroupInfo AP_Baro::var_info[] = {
 #endif
 #endif  // HAL_BARO_WIND_COMP_ENABLED
 
+#if AP_BARO_THST_COMP_ENABLED
+    // @Param: 1_THST_SCALE
+    // @DisplayName: Thrust compensation
+    // @Description: User provided thrust scaling in Pascals. This is used to adjust linearly based on the thrust output for local pressure difference induced by the props.
```

Comment:
Txs for this. I guess we could remove the "User provided" at the beginning of the Description.  I think most parameters are probably "user provided"

Reply:
Fixed